addon/mixins/copyable.js

```diff
@@ -137,12 +137,21 @@ export default Ember.Mixin.create({
           !PRIMITIVE_TYPES.includes(type)
       ) {
         let value = this.get(name);
-        let transform = getTransform(this, type, _meta);
 
-        // Run the transform on the value. This should guarantee that we get
-        // a new instance.
-        value = transform.serialize(value, attributeOptions);
-        value = transform.deserialize(value, attributeOptions);
+        if (
```

Comment:
This can just be 

`if (canInvoke(value, 'copy')) { ... }`

Reply:
Yes, we can use `Ember.canInvoke`. But i would like to be sure `value` is an Ember Object which extends `Ember.Copyable` mixin and so, `value#copy()` is really the method that copy this object (_and not a method to do anything else_). I do not know how to do... Perhaps do you know?

Reply:
@offirgolan Any response ? I will make the change :)